models/ed/R/other.helpers.ED2.R

```diff
@@ -2,9 +2,33 @@
 #' 
 #' @author Alexey Shiklomanov
 #' @inheritParams base::list.files
+#' @export
 list.files.nodir <- function(path, ...) {
     allfiles <- list.files(path, ...)
     dirs <- list.dirs(path, ...)
     outfiles <- setdiff(allfiles, dirs)
     return(outfiles)
 }
+
+
+#' Function translating pecan vars to ED vars
+#' var.names <- c("DBH", "AGB", "AbvGrndWood")
+#' @export 
```

Comment:
is this a function that is needed outside of the ED code?

Reply:
no..but if I don't export it I have to source it everytime while I'm developing the code. Even if I load  PEcAn.ED2 it can't find the function when it's called through other functions. Maybe I'm missing something?